### ISSUE TITLE
[HowToTest] How to Test: "Color Contrast", "Form & Inputs" pages need updates to accommodate dark mode settings

### ISSUE THREAD

**Issue by rnguyen253** [tmobile/a11yEngineer#387]
Problem: The pages within the [How to test](https://www.magentaa11y.com/#/how-to-test-criteria/type/overview) “Color Contrast” and “Focus & Inputs” have color contrast needs called out that “don’t pass” which do pass in dark mode. We should update what is present to show light and dark mode examples, or something that doesn’t meet color contrast in both settings.

On [Color Contrast](https://www.magentaa11y.com/#/how-to-test-criteria/type/color-contrast), we need to update (see “colorContrast_screenshotsToUpdate” screenshot attached)

the screenshots of our former MagentaA11Y site to the current site

“3. What to test for” to include light mode and dark mode examples because currently the “doesn’t pass” examples DO pass in dark mode. 

On [Focus & Inputs](https://www.magentaa11y.com/#/how-to-test-criteria/type/forms-&-inputs), we need to update (see “formsAndInputs_inaccessibleInput” screenshot attached)

The Radio Button example doesn’t have any radi button visual cues or ability to interact

The “✓ Ensure forms can receive keyboard focus and have focus indicators“ inaccessible option could be updated to be done better

Review mention of color contrast like “✓ Ensure forms meet color contrast requirements” because currently the “doesn’t pass” examples DO pass in dark mode
